accounts/Makefile

```diff
@@ -135,24 +135,9 @@ release-check:
 .PHONY: release-finish
 release-finish: release-copy release-check
 
-.PHONY: docs-copy
-docs-copy:
-	mkdir -p $(HUGO); \
-	mkdir -p $(HUGO)/content/extensions; \
-	cd $(HUGO); \
-	git init; \
-	git remote rm origin; \
-	git remote add origin https://github.com/owncloud/owncloud.github.io; \
-	git fetch; \
-	git checkout origin/source -f; \
-	rsync --delete -ax ../docs/ content/extensions/$(NAME)
-
-.PHONY: docs-build
-docs-build:
-	cd $(HUGO); hugo
-
-.PHONY: docs
-docs: docs-copy docs-build
+.PHONY: config-docs-generate
+config-docs-generate:
+#	go run github.com/owncloud/flaex >| ../docs/extensions/$(NAME)/configuration.md
```

Comment:
Can you write a ticket that we want to enable flaex later on for the extensions that don't have it yet?

Reply:
👍 done https://jira.owncloud.com/browse/OCIS-655